front/lib/amplitude/config.ts

```diff
@@ -0,0 +1,2 @@
+// This is a public API key, which will endup in the user's browser anyway.
+export const AMPLITUDE_PUBLIC_API_KEY = "6ba33096c77a939358f9c21e12d73592";
```

Comment:
All the tracked events are stored in the back-end. Can you shed some light on why it's necessary to have this specific key in the front-end?

Reply:
There is a PR in review for the quick guide, which will use Amplitude on the front-end side to measure how much time people are spending on the guide.